### PR TITLE
chore: add font-awesome in upgrade process

### PR DESCRIPTION
**Pull request by carloslancha**

### CODE REVIEW

File: packages/liferay-theme-tasks/lib/upgrade/7.1/upgrade.js [hunk 1]
```diff
@@ -20,6 +20,13 @@ module.exports = function(options) {
 	gulp.task('upgrade:dependencies', function(cb) {
 		lfrThemeConfig.removeDependencies(['liferay-theme-deps-7.1']);
 		lfrThemeConfig.setDependencies(devDependencies.default, true);
+		lfrThemeConfig.setDependencies(
+			{
+				'liferay-font-awesome':
+					devDependencies.optional['liferay-font-awesome'],
```
Comment (jbalsas) [inline packages/liferay-theme-tasks/lib/upgrade/7.1/upgrade.js]: what does `optional` mean here?

Comment (jbalsas) [reply packages/liferay-theme-tasks/lib/upgrade/7.1/upgrade.js]: and which version of `liferay-font-awesome` will we be adding?

Comment (carloslancha) [reply packages/liferay-theme-tasks/lib/upgrade/7.1/upgrade.js]: We're defining devDependencies versions in `devDependencies.js`, so they can be easy to find and change if necessary.

In the last pr adding the support for font-awesome on theme creation I divided devDependencies in `default` ones, which are always added to a new theme, and the `optional` ones, which are added depending on user prompt answers.

Are you thinking in a better way?

**Comment by carloslancha** [comment]
I'm ok with adding a prompt for this. @marcoscv-work what do you think?

File: packages/liferay-theme-tasks/lib/upgrade/7.1/upgrade.js (hunk 1 above)
Comment (jbalsas) [reply packages/liferay-theme-tasks/lib/upgrade/7.1/upgrade.js]: That's okay, I just wasn't aware of it 👍

File: packages/generator-liferay-theme/generators/app/templates/src/css/custom.css [hunk 2]
```diff
@@ -1,6 +1,12 @@
 /* These inject tags are used for dynamically creating imports for themelet styles, you can place them where ever you like in this file. */
 
 /* inject:imports */
+<% if (fontAwesome) { %>
+@import 'liferay-font-awesome/scss/font-awesome';
+@import 'liferay-font-awesome/scss/glyphicons';
+    
+@import 'clay/base';
```
Comment (jbalsas) [inline packages/generator-liferay-theme/generators/app/templates/src/css/custom.css]: Is this really needed? @marcoscv-work?

**Comment by jbalsas** [comment]
Hey @carloslancha, I've made some adjustments and sent https://github.com/liferay/liferay-js-themes-toolkit/pull/382. Could you please take a look so we can go over it?

**Pull request closed without merging**